lumapps/api/client.py

```diff
@@ -378,19 +374,19 @@ def get_call(self, *method_parts, **params):
     def iter_call(self, *method_parts, **params):
         """
         Args:
-            *method_parts (str): API method.
-            **params: Parameters.
+            *method_parts (List[str]): API endpoint (eg user/get or "user", "get").
+            **params (dict): Parameters of the call
 
         Yields:
-            dict: Objects returned by API method.
+            dict: An object returned as a response.
```

Comment:
Original phrasing was better.

Reply:
Api method is not the correct word, the method is GET, POST what is called is an api endpoint. So I would say that the correct wording would be `An object, or list of objects returned by the LumApps API endpoint` or `An object, or list of objects returned by the LumApps API`, what do you think of that